src/p_acs.cpp

```diff
@@ -7575,7 +7576,6 @@ doplaysound:			if (funcIndex == ACSF_PlayActorSound)
 		case ACSF_GetPlayerScore:
 			{
 				const ULONG ulPlayer = static_cast<ULONG> ( args[0] );
-
```

Comment:
Looks like this change accidentally slipped in. This code has nothing to do with what the patches are doing.

Reply:
I think this is fixed in your latest patch.